src/benchmarks/src/bench_utils.cpp

```diff
@@ -0,0 +1,174 @@
+#include <clipper/json_util.hpp>
+#include <clipper/logging.hpp>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <unordered_map>
+
+#include "include/bench_utils.hpp"
+
+using namespace clipper;
+
+namespace bench_utils {
+
+const std::string CIFAR_DATA_PATH_PROMPT =
+    "Enter a path to the CIFAR10 binary data set: ";
+const std::string NUM_THREADS_PROMPT =
+    "Enter the number of threads of execution: ";
+const std::string NUM_BATCHES_PROMPT =
+    "Enter the number of request batches to be sent by each thread: ";
+const std::string BATCH_SIZE_PROMPT =
+    "Enter the number of requests per batch: ";
+const std::string BATCH_DELAY_MILLIS_PROMPT =
+    "Enter the delay between batches, in milliseconds: ";
+const std::string BATCH_DELAY_MICROS_PROMPT =
+    "Enter the delay between batches, in microseconds: ";
+const std::string LATENCY_OBJECTIVE_PROMPT =
+    "Enter the latency objective, in microseconds: ";
+const std::string REPORT_DELAY_SECONDS_PROMPT =
+    "Enter the delay between reporting metrics, in seconds: ";
+const std::string REPORTS_PATH_PROMPT =
+    "Enter the path to the file for contain benchmark reports: ";
+const std::string REPORTS_PATH_VERBOSE_PROMPT =
+    "Enter the path to the file for contain verbose benchmark reports: ";
+const std::string POISSON_DELAY_PROMPT =
+    "Enter \"true\" if you want the delay between request batches to be drawn "
+    "from a "
+    "poisson distribution: ";
+const std::string MODEL_NAME_PROMPT =
+    "Enter the name of the model you want queried: ";
+const std::string MODEL_VERSION_PROMPT =
+    "Enter the version of the model you want queried: ";
+
+std::string _get_prompt(std::string var) {
```

Comment:
Unfortunately, it doesn't seem like there is a more elegant way to do string matching in C++. However, I think that it would be beneficial to define prompts for a specific benchmark in its own implementation file. This would reduce clutter.

 Towards this end, we should wrap these utility methods in a class with a constructor that accepts a mapping of `desired_vars` to prompt variables. `_get_prompt` can then reference this map.